packages/libs/src/sdk/api/generated/default/apis/TracksApi.ts

```diff
@@ -252,6 +323,38 @@ export class TracksApi extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * @hidden
+     * Inspect an mp3 track including Content-Length in the response.
+     * Gets the availability of a streamable MP3 file of a track,
+     */
+    async inspectTrackRaw(params: InspectTrackRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<void>> {
```

Comment:
Kind of a bummer that we don't get any types for the headers or anything 😞

I wonder if we should override the non-raw version in the custom `TracksApi` class and extract the relevant info there, nicely typed and everything

Reply:
I think it's reasonable to pass around a response object when we want it. Less to maintain and break with overriding it in the custom spot. That was already a little confusing to me around the other stream endpoint that's there.

Reply:
This is now handled because we're doing the normal flow of getting a json response from the inspect route!